abb_hardware_interface/CMakeLists.txt

```diff
@@ -70,6 +70,8 @@ install(
 #############
 
 if(BUILD_TESTING)
+  find_package(ament_cmake_gtest REQUIRED)
+  find_package(ros_testing REQUIRED)
```

Comment:
I'm not sure the `ros_testing` dep is needed here since no tests run for this package. Am I missing something?